Neos.Flow/Classes/Mvc/Controller/SimpleActionController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Neos\Flow\Mvc\Controller;
+
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Mvc\ActionResponse;
+use Neos\Flow\Mvc\Exception\NoSuchActionException;
+use Neos\Flow\SignalSlot\Exception\InvalidSlotException;
+
+/**
+ * Provides most direct access to our request/response abstraction,
+ * the request comes directly from the dispatcher and goes directly back to it.
+ *
+ * For helpers to facilitate throws, forwards, redirects:
+ * @see SpecialResponsesSupport
+ *
+ * Views or other processing needs to be added to your controller as needed,
+ * helpers will be suggested here as they become available.
+ */
+class SimpleActionController implements ControllerInterface
```

Comment:
ha as a new class we can finally avoid making mistakes concerning what api is and whats not :D

I guess extending is okay xD - no just joking - but then we should probably mark it `@api`, also as there is relatively little code in `processRequest` and `resolveActionMethodName`, we _can_ consider making both final, to limit api scope, as creating an own plain controller by implementing the `ControllerInterface` is now super simple.